snippets/countdown.liquid

```diff
@@ -10,6 +10,7 @@
 <div class="countdown-wrapper">
   <div class='message'>{{ section_settings.countdown_message }}</div>
   <div class="countdown" id="count-{{ id }}" data-date="{{ section_settings.countdown_day }}-{{ section_settings.countdown_month }}-{{ section_settings.countdown_year }}" data-time="{{ section_settings.countdown_hour }}:{{ section_settings.countdown_minute }}">
+    <ion-icon class="countdown-icon" name="time-outline"></ion-icon>
```

Comment:
Check the product page countdown and the tag item is no longer displaying as it should be.